runtimes/kotlin-1.6/src/main/kotlin/io/openruntimes/kotlin/Server.kt

```diff
@@ -28,9 +30,34 @@ suspend fun execute(ctx: Context) {
     val request = RuntimeRequest(ctx)
     val response = RuntimeResponse()
 
+    val outStream = ByteArrayOutputStream()
+    val errStream = ByteArrayOutputStream()
+    val userOut = PrintStream(outStream)
+    val userErr = PrintStream(errStream)
+    val systemOut = System.out
+    val systemErr = System.err
+
+    System.setOut(userOut)
+    System.setErr(userErr)
+
     try {
-        ctx.result(codeWrapper.main(request, response).data)
+        val userResponse = codeWrapper.main(request, response)
+        val output = mutableMapOf(
+            "response" to userResponse.data,
+            "stdout" to outStream.toString()
+        )
+        ctx.json(output)
```

Comment:
Doesn't the output need to be serialised?  We are doing it in Java, and the others

Reply:
@abnegate  helped resolve the comment 👍